signer/tests/integration/regtest.rs

```diff
@@ -369,3 +370,16 @@ pub fn p2tr_sign_transaction<U>(
 
     tx.input[input_index].witness = Witness::p2tr_key_spend(&signature);
 }
+
+pub fn generate_withdrawal() -> (WithdrawalRequest, Recipient) {
+    let recipient = Recipient::new(AddressType::P2tr);
+
+    let req = WithdrawalRequest {
+        amount: rand::rngs::OsRng.sample(Uniform::new(100_000, 250_000)),
+        max_fee: 250_000,
+        address: recipient.address.clone(),
+        signer_bitmap: Vec::new(),
+    };
```

Comment:
Personally, I prefer to have test utilities take an rng as input so that we can seed the rng during the tests to ensure the tests are deterministic.

Some developers like to have tests being non-deterministic, which can make sense because it expands the testing surface. However, in this scenario my preferred approach is to explicitly randomize the seed, log it and use a seeded rng for all test utilities - which ensures that we're able to reproduce any bugs that pop up in CI.

```suggestion
pub fn generate_withdrawal(rng: &mut impl rand::Rng) -> (WithdrawalRequest, Recipient) {
    let recipient = Recipient::new(AddressType::P2tr);

    let req = WithdrawalRequest {
        amount: rng.sample(Uniform::new(100_000, 250_000)),
        max_fee: 250_000,
        address: recipient.address.clone(),
        signer_bitmap: Vec::new(),
    };
```

Reply:
Yeah that's fair. And we can randomly generating the seed too. We can tackle something like that in a future PR. I created https://github.com/stacks-network/sbtc/issues/191 to track it.